site/reference/cli/fn/doc/README.md

```diff
@@ -31,8 +35,8 @@ print the error message to STDERR.
 <!--mdtogo:Examples-->
 
 ```shell
-# diplay the documentation for image gcr.io/kpt-fn/set-namespace:v0.1.1
-$ kpt fn doc --image gcr.io/kpt-fn/set-namespace:v0.1.1
```

Comment:
don't replace this valid example, add a new one.

Reply:
It would be confusing to show 2 examples. Why would users enter this prefix if they don't have to? I feel having one example with new format is cleaner.